src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -150,60 +161,71 @@ public void parse_compulsoryFieldMissing_failure() {
         assertParseFailure(parser, IC_DESC_BOB + NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB
```

Comment:
Shouldn't all the missing *field* prefix test also include `VALID_BLOOD_TYPE_BOB` so that the tests would be consistent with each other?